A/AmazonCACerts/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# This script downloads and bundles CA certificates from Amazon Web Services (AWS).
+# Right now only the RDS root certificate is included, but more could be added in the future.
+
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "AmazonCACerts"
+# Info here: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html
+certificate_version = "2019"
+tarball_version = v"1.0.0"
```

Comment:
We tend to use the same version number as the upstream "package", as we do in for `MozillaCACerts`: https://github.com/JuliaPackaging/Yggdrasil/blob/816c103b16c1b63fdb1ef17677be103c3efe3cb8/M/MozillaCACerts/build_tarballs.jl#L7-L8

Reply:
Oh, I must have done something wrong because I previously tried something similar and had an error, but not now.
Thanks, I'll update that.